app/controllers/projects_controller.rb

```diff
@@ -61,6 +61,14 @@ def destroy
     redirect_to admin_projects_path
   end
 
+  def deploy_group_versions
+    before = params[:before] ? Time.parse(params[:before]).to_s(:db) : Time.now.to_s(:db)
```

Comment:
I would assume that `Project#last_deploy_by_group` would take a `Time` as a parameter, rather than a string. Is there a reason you're transforming them into strings?